### ISSUE TITLE
Screen reader does not announce list information and position count for emojis buttons in windows.

### ISSUE THREAD

**Issue by msftedad** [facebook/react#36422]
**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26200.8246)
Browsers: Edge (Version 147.0.3912.98), Chrome (Version 148.0.7778.96)
Screen Reader: NVDA (2026.1), JAWS (2026.2603.86), Narrator
URL: https://ealush.com/emoji-picker-react/

**User Impact:**
Screen reader users cannot understand the total number of emojis available or their current position within the list. This makes navigation difficult and reduces efficiency while selecting emojis.

**Pre-Requisite:** 
1. Turn on Screen Readers:
NVDA: Ctrl + Alt + N
JAWS: Ctrl + Alt + J
Narrator: Ctrl + win + Enter
2. Verbosity:
NVDA: Default 
JAWS: Beginner, Highest 
Narrator: Default

**Repro Steps:**

1. Open the URL: https://ealush.com/emoji-picker-react/ in edge/chrome browser.
2. Emoji picker for React page will appear.
3. Press tab key to reach to the Emoji controls.
4. Verify whether screen reader announces list information and position count for emojis buttons or not.

**Actual Result:**
Screen reader does not announce list information and position count for emojis buttons in windows.
Screen reader announces: GrinningFace button

**Refer Attachment:**

1. Screen reader does not announce list information and position count for emojis buttons in windows.png
2. Screen reader does not announce list information and position count for emojis buttons in windows.mp4

https://github.com/user-attachments/assets/cc0ddcef-06d0-4e2d-968e-c5dafc879f56
<img width="1948" height="1324" alt="Image" src="https://github.com/user-attachments/assets/09d0c6a4-810e-4787-a49d-4a08109620a4" />

**Expected Result:**
Screen Reader should announce list information and position count for emoji button in windows.
Screen Reader should announce: List with 553 items Grinning face button 1 of 553

**Comment by ANU-2524** [comment]
Could you assign this issue to me?

**Comment by sophiebits** [comment]
This is not a bug in React.
